tasks/frontEnd/src/components/ReviewCard.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <Card style="margin-bottom: 25px;" v-if="displayAd">
```

Comment:
Nit: Inline styles

Reply:
For some reason if I applied a class name on the Card element it removed all the inbuilt styles, (rounded edges, border etc), so I left this inline. In a real world scenario I would investigate why this was the case and build a more suitable solution.